README.md

```diff
@@ -84,6 +84,10 @@ If a [refresh token](https://openid.net/specs/openid-connect-core-1_0.html#Refre
 
 Requests made to the `/logout` location invalidate both the ID token, access token and refresh token by erasing them from the key-value store. Therefore, subsequent requests to protected resources will be treated as a first-time request and send the client to the IdP for authentication. Note that the IdP may issue cookies such that an authenticated session still exists at the IdP.
 
+#### RP-Initiated OIDC Logout
```

Comment:
Minor nit. Since this is the first use of the acronym, we should define what it means. Maybe something like "Relying Party (RP) Initiated OIDC Logout" would be better.

Reply:
Thank you for the suggestion. To maintain consistency with established terminology and the OIDC spec "OpenID Connect RP-Initiated Logout," I prefer to keep "RP-initiated logout" as is. Additionally, a Google search for "RP-initiated logout" yields over 8k references, indicating it is a widely recognized term.